sci-biology/spm/spm-12_p6470.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit versionator
+
+MY_PV=$(get_major_version)
+
+DESCRIPTION="Analysis of brain imaging data sequences for Octave or Matlab"
+HOMEPAGE="http://www.fil.ion.ucl.ac.uk/spm/"
+SRC_URI="http://www.fil.ion.ucl.ac.uk/spm/download/restricted/eldorado/${PN}${MY_PV}.zip -> ${P}"
```

Comment:
Further, your downloading `spm12.zip`, but the update `spm12_updates_r6470.zip` from http://www.fil.ion.ucl.ac.uk/spm/download/spm12_updates/. isn't fetched. You should check, whether `spm12.zip` + the update is still needed or if the full source is provided with `spm12_updates_r6470.zip`.